Sources/Sharing/Documentation.docc/Extensions/AppStorageKey.md

```diff
@@ -14,10 +14,14 @@ test proves:
 
 ```swift
 @Test func externalWrite() {
-  @Shared(.appStorage("isOn")) var isOn = true
-  #expect(isOn == true)  
-  UserDefaults.standard.set(false, forKey: "isOn")
-  #expect(isOn == false)
+  withDependencies {
+    $0.defaultAppStorage = .liveValue
+  } operation: {
+    @Shared(.appStorage("isOn")) var isOn = true
```

Comment:
Could you update this to use the `@Test(.dependency(…))` trait? And also remove from the defaults at the top of the test so to have a clean slate?